demos/ids-tabs/index.yaml

```
@@ -15,3 +15,5 @@
     description: Showing a tab element side by side working with 4.x
   - link: standalone-css.html
     description: Showing limited tabs that works with css only
+  - link: header-tabs.html
+    description: Tabs within header and content in body updating.
```

```suggestion
    description: Header Tabs and content in body updating.
```

Just thinking should refer to it as header tabs as thats whats known